src/main/java/de/hbz/lobid/helper/Email.java

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2019 hbz, Pascal Christoph. Licensed under the EPL 2.0*/
+package de.hbz.lobid.helper;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Properties;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Multipart;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+
+/**
+ * Sends an email via localhost smtp. Make sure the server is configured
+ * properly.
+ * 
+ * @author Pascal Christoph (dr0i)
+ *
+ */
+public class Email {
+
+	/**
+	 * Sends an email.
+	 * 
+	 * @author Pascal Christoph (dr0i)
+	 * 
+	 * @param FROM_NAME the name before the "@" of an email
+	 * @param TO_EMAIL the complete email address of the receiver
+	 * @param SUBJECT the subject of the mail
+	 * @param MESSAGE the plain text message of the mail
+	 */
+	static public void sendEmail(final String FROM_NAME, final String TO_EMAIL,
+			final String SUBJECT, final String MESSAGE) {
```

Comment:
All caps is typically used for constants, not parameters. I'd suggest `fromName`, `toEmail`, `subject`, and `message`.